service/src/main/java/com/commercetools/pspadapter/payone/transaction/paymentinadvance/BankTransferInAdvanceChargeTransactionExecutor.java

```diff
@@ -12,6 +12,19 @@
 
 import static io.sphere.sdk.payments.TransactionType.CHARGE;
 
+/**
+ * <b>Note:</b>: according to <i>TECHNICAL REFERENCE PAYONE Platform Channel Server API v2.92</i>
+ * (chapter <i>3.2.2 Initiating payment process (authorization)</i>)
+ * and <i>Sequenzdiagramme20170130</i> (chapter <i>Retailer – Zahlart Vorkasse mit Vorautorisierung</i>)
```

Comment:
You refer to a document which is not uploaded this repo? Can we upload it?

Reply:
Unfortunately we can't: we already asked Payone support and they mentioned these docs are kind of internal and they supply them to the customers by request (as for _ Sequenzdiagramme_) or the customers could download them from merchant center (as for _TECHNICAL REFERENCE_)